src/annotator/anchoring/pdf.js

```diff
@@ -79,16 +79,32 @@ async function getPageView(pageIndex) {
     // a "pdfPage" property.
     pageView = await new Promise(resolve => {
       const onPagesLoaded = () => {
-        document.removeEventListener('pagesloaded', onPagesLoaded);
+        pdfViewer.eventBus.off('pagesloaded', onPagesLoaded);
         resolve(pdfViewer.getPageView(pageIndex));
       };
-      document.addEventListener('pagesloaded', onPagesLoaded);
+      pdfViewer.eventBus.on('pagesloaded', onPagesLoaded);
```

Comment:
This fixes a compatibility issue with new versions of PDF.js that we hadn't noticed before. PDF viewer events are no longer emitted as DOM events. It should be extracted into a separate PR and shipped first.